build.rs

```diff
@@ -5127,8 +5443,6 @@ mod c {
                     "arm/bswapsi2.S",
                     "arm/clzdi2.S",
                     "arm/clzsi2.S",
-                    "arm/comparesf2.S",
-                    "arm/divmodsi4.S",
```

Comment:
Was this deletion accidental perhaps? (or is the assembly weird like that?)

Reply:
Oh, oops.

Reply:
Fixed.